stylelint-polaris/tests/index.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable no-console */
+const fs = require('fs');
+const path = require('path');
+
+const stylelint = require('stylelint');
+
+const fixturePath = path.join(process.cwd(), 'tests/fixture.css');
+const fileContent = fs.readFileSync(fixturePath, 'utf-8');
+
+(async () => {
```

Comment:
Do we have support for top-level `await` in Node >14.8? We are on 16 so I think we could remove this IIFE

Reply:
I thought so as well. Maybe I didn't `dev up`. Let me try again.

Reply:
Seems like it's `.mjs` files only